health-checker/src/main/java/io/stargate/health/Server.java

```diff
@@ -0,0 +1,43 @@
+package io.stargate.health;
+
+import io.stargate.health.metrics.api.Metrics;
+import org.glassfish.hk2.utilities.binding.AbstractBinder;
+
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import io.dropwizard.Application;
+import io.dropwizard.configuration.ResourceConfigurationSourceProvider;
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.setup.Environment;
+
+public class Server extends Application<ApplicationConfiguration> {
+    private BundleService bundleService;
+    private final Metrics metrics;
+
+    public Server(BundleService bundleService, Metrics metrics) {
+        this.bundleService = bundleService;
+        this.metrics = metrics;
+    }
+
+    @Override
+    public void run(final ApplicationConfiguration applicationConfiguration, final Environment environment) {
+
+        environment.getObjectMapper().configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
+        environment.getObjectMapper().registerModule(new JavaTimeModule());
+
+        environment.jersey().register(new AbstractBinder() {
+            @Override
+            protected void configure() {
+                bind(bundleService).to(BundleService.class);
+            }
+        });
+        environment.jersey().register(CheckerResource.class);
+    }
+
+    @Override
+    public void initialize(final Bootstrap<ApplicationConfiguration> bootstrap) {
+        super.initialize(bootstrap);
+        bootstrap.setConfigurationSourceProvider(new ResourceConfigurationSourceProvider());
+        bootstrap.setMetricRegistry(metrics.getRegistry());
```

Comment:
Currently health-checker doesn't prefix its metrics, because what we pass here is both the registry where the module will publish its own metrics, and the one that is used to expose the `/metrics` endpoint (where we want every metric).

I'll probably need to switch to a custom servlet to switch to Prometheus format anyway, so I figured I'd address that later.

Reply:
Fixed in faee7d8.